src/community/security/oauth2-openid-connect/oauth2-openid-connect-core/src/main/java/org/geoserver/security/oauth2/OpenIdConnectFilterConfig.java

```diff
@@ -19,11 +19,14 @@ public class OpenIdConnectFilterConfig extends GeoServerOAuth2FilterConfig {
     String tokenRolesClaim;
     String responseMode;
     boolean sendClientSecret = false;
+    boolean allowBearerTokens = false;
```

Comment:
Since bearer authentication was already working although without validation of the token, maybe this flag could be kept as true by default.

Reply:
I changed it to true. I was a bit worried leaving bearer tokens as the default.